### ISSUE TITLE
Site design upgrade

### ISSUE THREAD

**Issue by stitzelj** [ficlatte/main#32]
I just want to put this here so it's kind of on the official to-do at some point.

Overall, I think the basic layout is good, as it has key callbacks to Ficly. I'm still planning to update the landing pages for Challenges and Prompts so they mirror the front page, unless folks just decide they like things the way they are now. I'd kind of like to break the list of challenges out into active and inactive ones, and maybe do popular and featured sections, as well.

How close to Ficly's design do we ultimately want to go with Ficlatte? Or do we just want to kind of strike out and do our own thing? 

Any suggestions for color schemes or other design elements?

And is it possible to get the original Photoshop/Illustrator/etc file for that banner graphic so we can make adjustments to it (particularly in size) as necessary?

Anything else we should be talking about as it relates to site design?

**Comment by ethel-t-frog** [comment]
I think that the site should be familiar to Ficly users, but I don't think we should try to ape the design too much. The guys at Viget Labs did that, and we shouldn't steal their hard work.  I'm happy to go with pretty much anything that looks good and works.

The current look and feel was the mastermind of Ficlatté user dkscully (a personal friend), and I think it works, but I agree that it is a bit plain and could use a bit more zazzle.  I am acutely aware that I am an electronic engineer to the core, however, and I have no aesthetic design skills at all.  Just run a potential new design past me before committing too much time to it and I'm pretty open to anything really.  I mean, you saw the original site design in two clashing shades of purple...

**Comment by stitzelj** [comment]
I started a branch on my fork for rebuilding the challenge and prompt pages and ended up killing two birds with one stone. I went ahead and gave the site a makeover while I was working, same basic layout just with some coffeeshop colors and updated graphics. I've included a screenshot of the challenges page below for your reference. I think it actually looks pretty snappy. I won't merge it into the main branch until @ethel-t-frog grabs the current PR (unless you guys would rather I just put it in there with everything else). If you want to take a look at the changes in the meantime, feel free to pull the branch for your review.

![screenshot from 2017-02-27 15-43-52](https://cloud.githubusercontent.com/assets/1470826/23379278/59afcdbc-fd04-11e6-94bb-8e71cbf48dfb.png)

**Comment by ethel-t-frog** [comment]
Looks gorgeous.

I'd prefer to merge the updates separately if that's OK.  Bear with me.  I will get onto it as soon as I can.

**Comment by stitzelj** [comment]
Take your time. I'm thinking about making my next project splitting the challenge and prompts modules off into their own apps. For sanity. The castle.views file is beginning to get a little unwieldy.

**Comment by ethel-t-frog** [comment]
Sounds good.  I've created issue #34 to cover this work.

**Comment by stitzelj** [comment]
I've been playing a lot with the site design the last couple of weeks, mostly because the more I play with it, the more I find little areas I've overlooked. The design is pretty polished at this point, and if there's anything I've overlooked by now, it's pretty minor. I even designed a new coffee bean badge for challenge winners and included a field in the author view for tallying up total wins (coffee beans earned). I think you'll like it.
